app/src/main/java/com/github/onlynotesswent/model/note/Note.kt

```diff
@@ -121,7 +130,13 @@ data class Note(
         return CommentCollection(
             mutableCommentsList.map {
               if (it.commentId == commentId)
-                  Comment(generateId(it.userId, content), it.userId, content)
+                  Comment(
```

Comment:
Seems wasteful to use a map, why not retrieve the index of the comment with `.indexOfFirst(it.commentId == commentId)`, then modify only that element. It is possible this is equivalent based on how these functions are implemented, but still seems a clearer implementation

Reply:
Implementation decision to be discussed later, need to consider possible collisions with commentID and what to do with it.